### PR TITLE
language about best-effort support

### PR DESCRIPTION
**Pull request by kingdonb**
Addresses these todos:

- clearly define what the limits of community support are
- which venue is appropriate for which enquiry

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -38,11 +38,26 @@ If all of this sounds like we're putting you off, that's not how this is meant.
...
+
+You can help ensure a higher quality of best-effort support by formulating inquiries thoughtfully and making a considerate effort to place them in the most appropriate venue.
+
+Here are some guidelines about which venue makes the most sense:
+
+| Form of inquiry                                                      | Venue                 |
+| -------------------------------------------------------------------- | --------------------- |
+| Something is not working as intended / I found a bug                 | Issue                 |
+| A feature is too limited for my \<use-case>                          | Discussion            |
+| I want Flux to be able to do \<x>                                    | Discussion (Proposal) |
```
Comment (hiddeco) [inline SUPPORT.md:55]:
```suggestion
| I want Flux to be able to do `<x>`                                    | Discussion ([Proposal](https://github.com/fluxcd/flux2/discussions/categories/proposals)) |
```

File: SUPPORT.md [hunk 2]
```diff
@@ -38,11 +38,26 @@ If all of this sounds like we're putting you off, that's not how this is meant.
...
+Similar to a best-effort network, the capacity for our community as a whole to provide quality support for community members, and a welcoming environment for all contributors, depends heavily on the grace and good behavior of individual community members.
+
+You can help ensure a higher quality of best-effort support by formulating inquiries thoughtfully and making a considerate effort to place them in the most appropriate venue.
+
+Here are some guidelines about which venue makes the most sense:
+
+| Form of inquiry                                                      | Venue                 |
+| -------------------------------------------------------------------- | --------------------- |
+| Something is not working as intended / I found a bug                 | Issue                 |
+| A feature is too limited for my \<use-case>                          | Discussion            |
```
Comment (hiddeco) [inline SUPPORT.md:54]:
```suggestion
| A feature is too limited for my `<use-case>`                          | Discussion ([general](https://github.com/fluxcd/flux2/discussions/categories/general))            |
```

File: SUPPORT.md [hunk 3]
```diff
@@ -38,11 +38,26 @@ If all of this sounds like we're putting you off, that's not how this is meant.
...
+| I want Flux to be able to do \<x>                                    | Discussion (Proposal) |
+| Something is not working, (but I am not sure if I am doing it right) | Discussion (Q&A)      |
```
Comment (hiddeco) [inline SUPPORT.md:56]:
```suggestion
| Something is not working, (but I am not sure if I am doing it right) | Discussion ([Q&A](https://github.com/fluxcd/flux2/discussions/categories/q-a))      |
```

File: SUPPORT.md [hunk 4]
```diff
@@ -38,11 +38,26 @@ If all of this sounds like we're putting you off, that's not how this is meant.
...
+
+Similar to a best-effort network, the capacity for our community as a whole to provide quality support for community members, and a welcoming environment for all contributors, depends heavily on the grace and good behavior of individual community members.
+
+You can help ensure a higher quality of best-effort support by formulating inquiries thoughtfully and making a considerate effort to place them in the most appropriate venue.
+
+Here are some guidelines about which venue makes the most sense:
+
+| Form of inquiry                                                      | Venue                 |
+| -------------------------------------------------------------------- | --------------------- |
+| Something is not working as intended / I found a bug                 | Issue                 |
```
Comment (hiddeco) [inline SUPPORT.md:53]:
```suggestion
| Something is not working as intended / I found a bug                 | [Issue](https://github.com/fluxcd/flux2/issues)                 |
```

File: SUPPORT.md [hunk 5]
```diff
@@ -38,11 +38,26 @@ If all of this sounds like we're putting you off, that's not how this is meant.
...
+
+[Best-effort delivery](https://en.wikipedia.org/wiki/Best-effort_delivery) is explained on Wikipedia in terms of a "**best-effort network**, [on which] all users obtain best-effort service. Under best-effort, network performance characteristics such as network delay and packet loss depend on the current network traffic load."
+
+Similar to a best-effort network, the capacity for our community as a whole to provide quality support for community members, and a welcoming environment for all contributors, depends heavily on the grace and good behavior of individual community members.
+
+You can help ensure a higher quality of best-effort support by formulating inquiries thoughtfully and making a considerate effort to place them in the most appropriate venue.
+
+Here are some guidelines about which venue makes the most sense:
+
+| Form of inquiry                                                      | Venue                 |
```
Comment (hiddeco) [inline SUPPORT.md:51]: Maybe add "quick question" as another form of inquiry, with the venue being "Slack".

Comment (kingdonb) [reply SUPPORT.md:51]: I added it at the bottom of the list

**Review by hiddeco — APPROVED** [review]
Thank you @kingdonb :grapes:

**Comment by hiddeco** [comment]
@dholbach I will let you merge as this merges into your WIP branch. :sparkles:

**Review by dholbach — APPROVED** [review]
Great work!

**Pull request merged**